recipe/meta.yaml

```diff
@@ -152,15 +152,26 @@ outputs:
         - ray.autoscaler
 
   - name: ray-dashboard
+    build:
+      script:
+        - cd python/ray/new_dashboard/client
+        - npm install
+        - npm ci
+        - npm run build
+        # not sure why this seems to get copied on windows but not linux...
```

Comment:
Adding this line where this comment is fixes it in my testing:
```
- mkdir -p $SP_DIR/ray/new_dashboard/client  # [not win]
```

Reply:
Yeah, I know I can create the target directory, but it seems like such a dirty hack for something that should already be covered by standard conda functionality... Anyway, will try that out for size